meta_package_manager/managers/opkg.py

```diff
@@ -0,0 +1,183 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016-2018 Kevin Deldycke <kevin@deldycke.com>
+#                         and contributors.
+# All Rights Reserved.
+#
+# This program is Free Software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+from __future__ import (
+    absolute_import,
+    division,
+    print_function,
+    unicode_literals
+)
+
+import re
+
+from boltons.cacheutils import cachedproperty
+
+from ..base import PackageManager
+from ..platform import LINUX
+
+
+class OPKG(PackageManager):
+
+    platforms = frozenset([LINUX])
+
+    requirement = '>= 1.0.0'
```

Comment:
Latest `opkg` version seems to be 0.4.1: http://git.yoctoproject.org/cgit/cgit.cgi/opkg/commit/?id=fdf77e6674bad59e60d0cb58812cf1669676950e

Can you at least change the required version to that that one?

Reply:
Oh, I overlooked this line, sorry. I corrected it.